### ISSUE TITLE
Cloud Credits Request: Git

### ISSUE THREAD

**Issue by therealmitchconnors** [cncf/credits#24]
### First and Last Name

Mitch Connors

### Email contact

mitchconnors@gmail.com

### Project

Istio

### The plan with the resources

Make an IDE easily accessible to new contributors

### Budget

???

### Provider

Azure

### Additional information

I'm interested in setting up codespaces for Istio contributors, but am currently limited by the features offered to the free tier.  Is there a way to have credits for maintainers to quickly spin up a dev env?

**Comment by caniszczyk** [comment]
@therealmitchconnors we are still working out what codespaces access we will provide to CNCF projects on top of what we have currently. We do currently have a relationship with Gitpod where it's available in an unlimited fashion to CNCF projects: https://www.gitpod.io/vs/github-codespaces - they are very similar in spirit tools

**Comment by therealmitchconnors** [comment]
Thanks for clarifying.  FWIW, the Istio Community chose codespaces over gitpods for two reasons: 1) the devcontainer standard is open, and allows substantially more flexibility that gitpods (the gitpod standard requires substantial hacking to get istio building correctly), and 2) codespaces can be spun up by vscode locally with the click of a button, where gitpod requires an entire kubernetes cluster to run locally.  

We may add gitpod config if others in the community are interested, but for now I think we will stick to codespaces - even if that means paying for our own compute...

**Comment by addikt1** [comment]
@therealmitchconnors I'm also interested in codespaces for cncf landscape. have you or @caniszczyk considered devcontainers since they would concievable work with codespaces, and run locally with a Docker runtime? The fact that multiple repos are needed to build a custom landscape, plus (my, anyway) need to cache images locally, makes for a large, semi-complicated setup just to do a local build and codespace-type of solution would be ideal.

**Comment by therealmitchconnors** [comment]
Yes, the Istio project is in the process of rolling out codespaces to all our repos, and the codespaces configuration is compatible with dev containers, so contributors can choose local or remote environments with consistent results.  GitHub has also graciously granted Istio free prebuild storage, to make the process smoother.  

We would love to have some compute credits for maintainers, but for now we are each paying for our own codespaces (my bill was less than $2 last month).
